confluent/docker_utils/cub.py

```diff
@@ -229,6 +230,37 @@ def check_kafka_rest_ready(host, port, service_timeout):
         return False
 
 
+def check_connect_ready(host, port, service_timeout):
+    """Waits for Connect to be ready.
```

Comment:
hmm, guess it makes sense to be consistent given other docstrings, but sort of weird here since this doesn't follow standard python [docstring format](https://www.python.org/dev/peps/pep-0257/)

Reply:
@ewencp I was going for total consistency.  Fixing docstring for all this code could be a separate jira/PR

Reply:
sure, that makes sense. just calling it out since I noticed it :)